### DISCUSSION TITLE
Thermostat support

### DISCUSSION THREAD

**Discussion by Denny5667** [mongoose-os-apps/shelly-homekit#1481] [Ideas]
It would be great if you could add thermostat feature support to Shelly 1/1PM. 
As Shelly supports up to three Dallas temperature sensors, I would suggest to calculate mean temperature. A hysteresis is also required to avoid frequent switching of relay.

Thanks in advance!

**Comment by andykearney** [comment]
+1 for this. Currently doing it via a Sonoff TH and HAA but it’s a big unit to hide away for me. The Shelly option would be far superior.

**Comment by pfeifix** [comment]
Any news about when the HK thermostat support will be available for the Shelly 1?

**Comment by pfeifix** [comment]
Any update?
